code/src/components/EndPage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Bar} from 'components/Form'
+
+export const EndPage = (props) => {
+
+ 
+    return (
+        <section className ="myForm">
+            <h1>Thank You</h1>
+            <div className = "result">
+    <p> Your name is: </p><p>{props.name}</p>
+    <p> The appartment size is: </p><p>{props.room}</p>
+    <p>Parking: </p>
+    {props.parking &&  <p>With Parking</p>}
+    {(props.parking === false) && <p>No parking</p> }
+    <p> Your city is: </p><p>{props.city}</p>
```

Comment:
Do you need two paragraphs here?
```suggestion
    <p>Your city is: {props.city}</p>
```